samples/fluent.js

```diff
@@ -42,15 +42,15 @@ var report = function (err, req) {
         referrer: req.header('Referer'),
         userAgent: req.header('User-Agent'),
         remoteIp: req.ip,
-        responseStatusCode: 500
-      }
-    }
+        responseStatusCode: 500,
+      },
+    },
   };
   structuredLogger.emit('errors', payload);
 };
 
 // Handle errors (the following uses the Express framework)
-app.use(function (err, req, res, next) {
+app.use(function(err, req, res) {
```

Comment:
@jmdobry I removed the `next` to be compliant with the linter. Good or bad idea?

Reply:
If `next` isn't used, nix it